extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -428,6 +428,8 @@ public void setSource(Source source) {
         /**
          * Force 'https' as the 'redirect_uri' parameter scheme when running behind an SSL terminating reverse proxy.
          * This property, if enabled, will also affect the logout `post_logout_redirect_uri` and the local redirect requests.
+         * 
+         * If enabled the oidc cookies will be created with secure=true.
```

Comment:
@mqs24d thanks for this PR. The only minor suggestion I have here is to tweak this doc line a bit.
The previous line I already use `if enabled`. So may be we can collapse the docs somehow but also expand a bit, may be drop '`if enabled`', for example
```
This property will also affect the logout `post_logout_redirect_uri` and the local redirect URIs. Additionally, the state, session and post logout cookies will have their 'secure' parameter set to 'true'.
```

What do you think ? if you agree then please squash the commits